src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+
+Depending on how your page handles the streamed content, you may want to wait
+until your scripts have loaded and are ready to process the stream. Preconnect
+helps you cut the waiting time to a single roundtrip once you’re ready to
+start fetching.
+
+## Prefetch
+
+`<link rel="prefetch">` is somewhat different from `<link rel="preload">` and
+`<link rel="preconnect">`, in that it doesn’t try to make something critical
+happen faster; instead, it tries to make something non-critical happen earlier,
+if there’s a chance.
+
+It does this by informing the browser of a resource that is expected to be
+needed as part of a future navigation or user interaction, i.e., something that
+*might* be needed later, if the user takes the action we’re expecting. These
+resources will be fetched at the **Lowest** priority in Chrome, when the current
+page is done loading and there’s bandwidth available.
+
+This means that `prefetch` is most suitable to preempt what the user might be
+doing next, and prepare for it, such as retrieving the first product details
+page in a list of results, or retrieving the next page in paginated content.
+
+    <link rel="prefetch" href="page-2.html">
+
+Bear in mind that prefetch doesn’t work recursively, though. In the example
+above you’d only be retrieving the HTML; any resources that `page-2.html` needs
+would not be downloaded ahead of time unless you explicitly prefetch them as
+well.
+
+### Prefetch Doesn't Work as an Override
+
+It’s important to note that you can’t use `<link rel="prefetch">` as a way of
+lowering the priority of an existing resource. In the following HTML, you might
+think that declaring `optional.css` in a prefetch would lower its priority for
+the subsequent `<link rel="stylesheet">`:
+
+    <html>
+      <head>
+        <link rel="prefetch" href="optional.css">
+        <link rel="stylesheet" href="optional.css">
+      </head>
+      <body>
+        Hello!
+      </body>
+    </html>
+
+However, this will actually cause your stylesheet to be fetched twice (albeit
+with a potential cache hit on the second one), once at the default **Highest**
+priority, and once at the **Lowest** priority, as prefetch kicks off a separate
+fetch:
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1374px; --aspect-ratio-w: 1374; --aspect-ratio-h: 190">
+    <img src="images/res-prio-prefetch.png"
+         alt="A screenshot of Chrome Developer Tools showing optional.css being
+              fetched twice">
+  </div>
+</figure>
+
+Double-fetching can be pretty bad for the user, as in this case not only would
+they have to wait for the render-blocking CSS, they would also potentially have
+their bandwidth wasted by downloading the file twice — and remember their
+bandwidth may very well be metered. Be sure to analyze your network requests
+thoroughly, and watch out for any double-fetching!
+
+## Other Techniques and Tools
+
+These three `<link>` types (and the bonus `<link rel="dns-prefetch">`) offer a
```

Comment:
Suggestion to repeat the names of the link types.

Reply:
Done, replaced "These three link types" with the actual link types.